### ISSUE TITLE
Missing alt tags for images

### ISSUE THREAD

**Issue by Ohannain** [frozolotl/muchpdf#13]
When compiling the document with --pdf-standard="ua-1" following error occurs:

```bash
[10:32:56] compiled with errors

error: PDF/UA-1 error: missing alt text
   ┌─ @preview/muchpdf:0.1.2\lib.typ:76:6
   │
76 │       image(page, format: "svg", ..args)
   │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   │
   = hint: make sure your images and equations have alt text
```

Alt tags are required by the UA/1 standard out of accessibility needs. Addition of alt tags for the generated images is welcomed, as the EU Accessibility Act requires all documents to be accessible since June 2025.

**Comment by frozolotl** [comment]
I'm sorry, I kind of forgot to answer here.

You can provide an alternative description just as you would with the `image` function. In fact, all `image` parameters should work just the same.

```typ
#muchpdf(read("image.pdf", encoding: none), alt: "A dolphin jumping through a hoop")
```
